modules/openapi-generator/src/main/resources/ruby-client/configuration.mustache

```diff
@@ -199,13 +199,27 @@ module {{moduleName}}
           },
 {{/isApiKey}}
 {{#isBasic}}
-        '{{name}}' =>
-          {
-            type: 'basic',
-            in: 'header',
-            key: 'Authorization',
-            value: basic_auth_token
-          },
+{{^isBasicBearer}}
+            '{{name}}' =>
```

Comment:
This indent seems to be wrong. Can you fix it?
https://github.com/OpenAPITools/openapi-generator/pull/2856/files#diff-c6b80b9611c8982577f1c2ceaeacd8e5R213